.github/workflows/RELEASE_ISSUE.yml

```diff
@@ -110,4 +110,4 @@ jobs:
       env:
         # Release manager is either the assignee from the newly created issue or the new assigned from the `assigned` trigger
         RELEASE_MANAGER: ${{ needs.create_release_issue.outputs.assignee || github.event.issue.assignee.login }}
-        ISSUE_LINK: https://github.com/${{ github.repository }}/issues/${{ fromJson(needs.create_release_issue.outputs.issue).number }}
+        ISSUE_LINK: https://github.com/${{ github.repository }}/issues/${{ needs.create_release_issue.outputs.issue && fromJson(needs.create_release_issue.outputs.issue).number || github.event.issue.number }}
```

Comment:
This assumes that the output is always provided, independent of [whether an action executed](https://github.com/camunda/camunda-modeler/pull/4884/files#diff-53dee1d30ef3e7583f7b1dfca3e88eb40d89a02590af290f0824516e62afd238L9), is that correct? Looks reasonable to me, i.e. we default to the current issue ("that got assigned") when no new issue was created.

Reply:
If an action is skipped, the output is empty but can still be part of condition: https://github.com/barmac/test/actions/runs/13677206229/job/38240292499